assets/css/amp-post-meta-box.css

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
It seems overkill to enqueue a file for this small amount of CSS but it is done that way in preparation of #709.